README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+This should be written as a model method because, though it could be written in the controller, it helps with separation of concerns to keep the controller file limited to things that are necessary such as variable declarations.
```

Comment:
I can see why you chose the model method, but the answer we were looking for is a `Helper Method`. These are intended to help render HTML based on conditions, while `Model Methods` are concerned with how data is going and coming from the database